src/toxcore/tcp/server/server_ext.rs

```diff
@@ -276,140 +321,94 @@ mod tests {
         format!("{}", ConnectionError::IncomingError {
             error: IoError::new(IoErrorKind::Other, "io error"),
         });
-        format!("{}", ConnectionError::ServerHandshakeError {
```

Comment:
ServerHandshakeError -> Timeout & IoError. EIther we should get rid of this test at all or fix it according to the new structure.

Reply:
There's no way to create an instance of Elapsed in tokio.

Reply:
Me neither.